src/assets/languages/english.json

```diff
@@ -3,14 +3,15 @@
     "registration": {
       "messages": {
         "heading": "Sign Up",
-        "buttonMessage": "Sign Up"
+        "registerButtonMessage": "Register",
+        "loginButtonMessage": "Back to Login"
       },
       "placeHolders": {
-        "firstName": "First Name",
-        "lastName": "Last Name",
-        "email": "Email",
-        "password": "Password",
-        "confirmPassword": "Confirm Password"
+        "firstName": "Your first name",
```

Comment:
The placeholders can be as it was earlier, didn't require any changes.
@Buddhiprabha @paavininanda @shreyans29 what do you guys think?

Reply:
Actually, these are not same as conventional placeholders. When the fields are not focused, what appears is called 'label'(in their terminology). So I didn't change them. I just added what emerges when we focus on the text field. I thought this might increase interactivity. More views welcomed :smile:

Reply:
If the new placeholders look good with the new design, i don't mind changing. I don't mind it being the same either. One is descriptive other one is just to the point. Don't have any preferences here.